docs/options.md

```diff
@@ -42,6 +42,13 @@ variables:
   CIBW_TEST_COMMAND: "nosetests {project}/tests"
 ```
 
+> .github/workflows/*.yml ([docs](https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables)) (can be global, in job, or in step)
+```yaml
+env:
+  CIBW_TEST_REQUIRES: nose
+  CIBW_TEST_COMMAND: "nosetests {project}/tests"
```

Comment:
```suggestion
  CIBW_TEST_COMMAND: nosetests {project}/tests
```

Quotes are technically not needed here.

Reply:
Also, shouldn't we change the test command example to pytest?

Reply:
Quotes aren't needed, but I think they're useful to people unfamiliar to YAML. But yes, pytest would probably be more familiar to most readers. If you change, could you change all the CI platform examples?